.github/workflows/stackhpc-all-in-one.yml

```diff
@@ -158,13 +158,15 @@ jobs:
           for attempt in $(seq 5); do
               if terraform apply -auto-approve; then
                   echo "Created infrastructure on attempt $attempt"
-                  break
+                  exit 0
               fi
               echo "Failed to create infrastructure on attempt $attempt"
               sleep 10
               terraform destroy -auto-approve
               sleep 60
           done
+          echo "Failed to create infrastructure after $attempt attempts"
```

Comment:
$attempt is referenced here outside the loop. Does it persist?

Reply:
I've now done what I should've done in the first place and just tested it. It does indeed work